drivers/atwinc15x0/doc.txt

```diff
@@ -156,10 +156,13 @@ Symbol                         | Default          | Description
 
 At the moment only WPA2 Personal Mode is supported. The required settings are:
 
-Parameter | Default   | Description
-:---------|:----------|:------------
-WIFI_SSID | "RIOT_AP" | SSID of the AP to be used.
-WIFI_PASS | -         | Passphrase used for the AP as clear text (max. 64 chars).
+Parameter  | Default   | Description
+:----------|:----------|:------------
+#WIFI_SSID | "RIOT_AP" | SSID of the AP to be used.
+#WIFI_PASS | -         | Passphrase used for the AP as clear text (max. 64 chars) [1].
```

Comment:
I'd suggest to migrate these values to include the `CONFIG_` prefix

Reply:
My concern is that user applications may have already used these documented configuration settings. We can change them but we should have some deprecation mechanism which. My approach was to use it in that way
```
#ifndef WIFI_SSID
#ifndef CONFIG_WIFI_SSID
#define WIFI_SSID       "RIOT_AP"
#else /* CONFIG_WIFI_SSID */
#define WIFI_SSID       CONFIG_WIFI_SSID
#endif /* CONFIG_WIFI_SSID */
#endif /* WIFI_SSID */
```
which allows to use both definitions.

Reply:
Sure, we should add a deprecation warning if we change them. It already has been done with other configurations (e.g., https://github.com/RIOT-OS/RIOT/pull/13129).